subworkflows/local/bgc.nf

```diff
@@ -43,9 +43,10 @@ workflow BGC {
 
         } else {
 
-            ch_css_for_antismash = "https://github.com/nf-core/test-datasets/raw/modules/data/delete_me/antismash/css.tar.gz"
-            ch_detection_for_antismash = "https://github.com/nf-core/test-datasets/raw/modules/data/delete_me/antismash/detection.tar.gz"
-            ch_modules_for_antismash = "https://github.com/nf-core/test-datasets/raw/modules/data/delete_me/antismash/modules.tar.gz"
+            // May need to update each version of antismash-lite due to changes to scripts inside these tars
```

Comment:
Why is there this huge number in the path now replacing `modules/`?

Reply:
I've added the hash because this tar.gz contain certain scripts required by AntiSmashLite, but this scripts are version dependent. To allow backwards compatibility testing of older versions, we use the hash of versions of the TAR files to ensure the contents of the TAR corresponds to the version of the tool (if that makes sense)